### DISCUSSION TITLE
【提案】3 是否支持引入链上证书收费模式？

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#739] [Polls]
### 提案：引入链上证书的收费模式

自2022年成立以来，WTF Academy一直致力于为开发者提供高质量的免费开源的Web3教育资源。我们的[WTF Solidity](https://github.com/AmazingAng/WTF-Solidity)和[WTF Ethers](https://github.com/WTFAcademy/WTF-Ethers)教程已经在GitHub上分别收获了超过1万和近3000的star，这离不开大家的支持和贡献。

为了确保我们能持续创作教程内容，迭代[wtf.academy教学平台](https://wtf.academy)，并更好的激励贡献者，我们计划引入一种新的收入模式——通过链上证书（SBT）实施微小的收费，新的SBT合约预计将部署在Base链。

这一模式参考了Coursera的证书收费策略，即：

- **教程和测试习题仍然免费提供**，所有人都可以自由学习；
- **链上证书（SBT）需要支付一定的费用（最低捐款额）**，以获取正式的认证。对于WTF Solidity课程，获得SBT的最低捐款额将设置在 $20~30；
- **所有WTF Academy的贡献者可以免费获得这些证书**，以感谢他们的贡献。

此举旨在为我们的Web3开源教育项目带来可持续的发展资金，同时保持我们对开源教育的承诺不变。所有收入将用于教程的更新、平台维护以及贡献者的奖励。

我们希望通过社区投票来决定这一提案的执行。大家可以把想法和建议写在回复中，期待听到每一位社区成员的声音！

截止日期：2024年6月10日24点。

**Comment by jeasonstudio** [comment]
可以提一下未来给证书 holder 提供更多权益（比如空投、使用内测功能等）。
另外证书的 tokenUri 内容也可以搞成动态的，完成越多的测试内容等级越高，图片越酷炫。

**Reply by AmazingAng** [reply]
必须酷炫

**Comment by keyneszeng** [comment]
支持，并且组建拥有SBT成员社群，不定期组织各项技术攻坚活动，带领活跃的社群成员获得更多的收益

**Comment by jes16jupyter** [comment]
May be we can also propose security services, like auditing... @AmazingAng

**Reply by AmazingAng** [reply]
我感觉直接提供审计服务有难度，咱们可以提供安全方向的培训，组织定期参与code4rena的比赛。

**Reply by jes16jupyter** [reply]
Agree. Count me in.

**Comment by Yinboan** [comment]
好奇为什么是base链？sol名声是被土狗搞臭了咩？嘿嘿嘿

**Reply by AmazingAng** [reply]
Optimism给WTF Academy的赞助最多，Base是OP superchain中最有潜力的一个。

**Comment by zhengqian8958** [comment]
如果在base链的话是不是可以考虑跟warpcast联动一下？好比可以提供一个frame给拿到证书的小伙伴的头像上加点装饰

**Reply by AmazingAng** [reply]
之后可以考虑的 先把证书弄好再说

**Comment by qnbh123** [comment]
以前认证过的会追发吗

**Reply by AmazingAng** [reply]
在测试网mint的过的课程证书，可以免费mint正式版sbt（同一课程）。

**Comment by thurendous** [comment]
非常有趣。支持

**Comment by calmanzeng** [comment]
支持

**Comment by Jamie-Jan** [comment]
支持!
